imap_data_access/file_validation.py

```diff
@@ -621,6 +624,13 @@ class SPICEFilePath(ImapFilePath):
         r"e(?P<version>\d{2})\."
         r"(?P<type>mk)"
     )
+    earth_attitude_pattern = (
+        r"earth_"
+        r"(?P<start_date>\d{6})_"
+        r"(?P<end_date>\d{8})_"
+        r"(?P<prediction_start_date>\d{8})"
```

Comment:
NAIF says that the filename pattern is: `earth_000101_yymmdd_yymmdd.bpc` which would mean that you want:
```suggestion
        r"(?P<end_date>\d{6})_"
        r"(?P<prediction_start_date>\d{6})"
```